pycbc/sensitivity.py

```diff
@@ -164,49 +168,43 @@ def volume_montecarlo(found_d, missed_d, found_mchirp, missed_mchirp,
         if max_param is not None:
             max_distance = max_param
         else:
-            # if no max distance given, use max distance actually injected
+            # If no max distance given, use max distance actually injected
             max_distance = max(found_d.max(), missed_d.max())
     else:
         raise NotImplementedError("%s is not a recognized parameter"
                                   % limits_param)
 
-    # volume of sphere
+    # Calculate volume of sphere out to maximum distance
     montecarlo_vtot = (4. / 3.) * numpy.pi * max_distance**3.
 
-    # arrays of weights for the MC integral
+    # Calculate weights based on distribution parameters
```

Comment:
```suggestion
    # Calculate weights for the MC integral
```
'MC integral' is useful scientific/statistical context to understand what is happening